.github/scripts/setup-env.sh

```diff
@@ -14,23 +14,26 @@ case $(uname) in
   Darwin)
     OS_TYPE=macos
     ;;
+  MSYS*)
+    OS_TYPE=windows
+    ;;
   *)
     echo "Unknown OS type:" $(uname)
     exit 1
     ;;
 esac
 
-echo '::group::Uninstall system JPEG libraries on macOS'
-# The x86 macOS runners, e.g. the GitHub Actions native "macos-12" runner, has some JPEG libraries installed by default
-# that interfere with our build. We uninstall them here and use the one from conda below.
 if [[ "${OS_TYPE}" == "macos" && $(uname -m) == x86_64 ]]; then
+  echo '::group::Uninstall system JPEG libraries on macOS'
```

Comment:
This just moves the grouping inside the `if` so this doesn't show up empty on non-macos runners.